### PR TITLE
Bankers algorithm in C

### PR DESCRIPTION
**Pull request by Karthik-Nayak98**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 

- Revert back the changes made on the C Style Guide. 
- Push all your functional code in a separate function and use main() for driver code

**Comment by Karthik-Nayak98** [comment]
Sure @HarshCasper

File: C/operating_system/bankers.c [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+4 3 3
+3 2 2
+9 0 2
+7 5 3
+1 1 2
+
+The safe sequence is:
+P2->P5->P1->P3->P4
+
+*/
```
Comment (HarshCasper) [inline C/operating_system/bankers.c]: Add a new line at the EOF

File: C/operating_system/bankers.c [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+    printf("Enter the maximum demand of resources by each process\n");
+    for(i = 0; i < p; ++i){
+        for(j = 0; j < r; ++j){
+            scanf("%d", &max[i][j]);
+        }
+    }
+
+    // This matrix is used to keep track of the executed process.
+    // Initializing with value '0' which means no process is executed yet.
```
Comment (HarshCasper) [inline C/operating_system/bankers.c]: Please use Multi-Line Comments

File: C/operating_system/bankers.c [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    int i;
+    printf("The safe sequence is:\n");
+    for(i = 0; i < p - 1; ++i)
+        printf("P%d->", seq[i]);
+    printf("P%d\n", seq[p-1]);
+}
+
+int main() {
+
+    int p, r, i, j;
```
Comment (HarshCasper) [inline C/operating_system/bankers.c]: Can we use more meaningful variable names for non-iterators?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Great work! Few changes requested otherwise good to merge :tada:

**Comment by Karthik-Nayak98** [comment]
Made all the changes mentioned above @HarshCasper.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**